common/src/test/scala/objects/ResourceSiloTest.scala

```diff
@@ -0,0 +1,332 @@
+// Copyright (c) 2017 PSForever
+package objects
+
+import akka.actor.{Actor, Props}
+import akka.routing.RandomPool
+import akka.testkit.TestProbe
+import base.ActorTest
+import net.psforever.objects.guid.TaskResolver
+import net.psforever.objects.{Avatar, GlobalDefinitions, Player}
+import net.psforever.objects.serverobject.resourcesilo.{ResourceSilo, ResourceSiloControl, ResourceSiloDefinition}
+import net.psforever.objects.serverobject.structures.{Building, StructureType}
+import net.psforever.objects.zones.Zone
+import net.psforever.packet.game.{PlanetSideGUID, UseItemMessage}
+import net.psforever.types.{CharacterGender, CharacterVoice, PlanetSideEmpire, Vector3}
+import org.specs2.mutable.Specification
+import services.ServiceManager
+import services.avatar.{AvatarAction, AvatarService, AvatarServiceMessage}
+
+import scala.concurrent.duration._
+
+class ResourceSiloTest extends Specification {
+  "Resource Silo" should {
+    "define" in {
+      val obj = new ResourceSiloDefinition
+      obj.ObjectId mustEqual 731
+    }
+
+    "construct" in {
+      val obj = ResourceSilo()
+      obj.Definition mustEqual GlobalDefinitions.resource_silo
+      obj.MaximumCharge mustEqual 1000
+      obj.ChargeLevel mustEqual 0
+      obj.LowNtuWarningOn mustEqual 0
+      obj.CapacitorDisplay mustEqual 0
+      //
+      obj.ChargeLevel = 50
+      obj.LowNtuWarningOn = 25
```

Comment:
I have no idea why I made this an Int (I'm going to claim temporary loss of sanity). It's supposed to be a Boolean for if the warning is on or off.

I can't push to this PR so after it's merged I'll fix it and submit another PR.